src/azul/terraform.py

```diff
@@ -800,11 +800,32 @@ def tf_config(self, app_name):
         # Setting this property using AWS API Gateway extensions to the OpenAPI
         # specification works around this issue.
         #
+        # We also use AWS API extensions to set the API Gateway default
+        # responses since, unlike Terraform, it automatically triggers the
+        # deployment of the REST API when the Gateway responses are added and/or
+        # modified.
+        #
         rest_api = resources['aws_api_gateway_rest_api'][app_name]
         assert 'minimum_compression_size' not in rest_api, rest_api
+        assert 'aws_api_gateway_gateway_response' not in resources, resources
         openapi_spec = json.loads(locals[app_name])
         key = 'x-amazon-apigateway-minimum-compression-size'
         openapi_spec[key] = config.minimum_compression_size
+        openapi_spec['x-amazon-apigateway-gateway-responses'] = {
+            f'DEFAULT_{response_type}': {
+                'responseParameters': {
+                    f'gatewayresponse.header.{k}': v
+                    for k, v in [
```

Comment:
Using a dictionary literal saves two characters per entry and makes it obvious that the keys are unique.